packages/cloud-cognitive/src/components/Datagrid/styles/useColumnRightAlign.scss

```diff
@@ -25,5 +25,6 @@
 }
 
 .#{$block-class}__right-align-cell-renderer.sortDisabled {
-  padding-right: 0;
+  /* stylelint-disable-next-line carbon/layout-token-use */
+  padding-right: 12px;
```

Comment:
I don't think there should be any changes to this file at all. The right aligned columns was already working, right?

Reply:
Removed the lines that I added, and added the single, 'padding-right: 0;' line.